cmd/node/main.go

```diff
@@ -360,8 +364,8 @@ func getSuite(config *config.Config) (crypto.Suite, error) {
 }
 
 func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
-	logLevel := ctx.GlobalString(logLevel.Name)
-	log.SetLevel(logLevel)
+	logLevelFlagValue := ctx.GlobalString(logLevel.Name)
```

Comment:
delete flagValue, no need.

Reply:
I changed this in order to remove GoLand's warnings